libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -694,13 +696,14 @@ constructor(
                 .navigatorPredictionMillis(DEFAULT_NAVIGATOR_PREDICTION_MILLIS)
                 .distanceFormatter(distanceFormatter)
 
-            // TODO provide a production routing tiles endpoint
-            val tilesUri = URI("")
-            val tilesVersion = ""
+            val tilesUri = URI("https://api.mapbox.com")
+            // Latest version available https://api.mapbox.com/route-tiles/v1/versions?access_token=
+            // as of 05/18/2020
+            val tilesVersion = "2020_02_02-03_00_00"
```

Comment:
did we decide we want to define this dated version here? 

seems like we still have a TODO, and that is to remove this

Reply:
As of today this is the latest tiles version available. This can always be overridden externally providing a different newest version via `MapboxOnboardRouterConfig` (if a new tiles version is added afterwards). BTW this is going to change eventually with the new Offline v2 so should be enough for unblocking the upcoming release.

Reply:
ok fair enough, v1 is to have working tiles. v2 has tile versions behind an api of some kind

is there an issue for that project? can we link it here?